docs/translations/translations.json

```
@@ -86,6 +109,7 @@
   "skipToContent": "Skip to content",
   "sourceCode": "Source code",
   "spacingUnit": "Spacing unit",
+  "spreadHint": "Any other props supplied will be provided to the root element ({{spreadHintElement}}).",
```

What about we give more context to the translation keys so we can better find them we work on the content (find orphans for instance, or navigate the file)? Maybe API as a prefix?
```suggestion
  "api.spreadHint": "Any other props supplied will be provided to the root element ({{spreadHintElement}}).",
```

---

Makes sense. It would be good if we had an automated way to check for orphans.

---

https://github.com/oliviertassinari/i18n-extract

---

These days, I was planning changes to propose something related to the i18n API. Based on an old @eps1lon  pull request. I was separating that way.

https://github.com/jaironalves/material-ui/blob/docs/api-i18n/docs/api/translations/components/Accordion/Accordion.json


I was at the very beginning of the idea, apprehensive how crowdin would give more context.